src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -1377,6 +1376,22 @@ private fun checkConstGenerics(holder: RsAnnotationHolder, constParameter: RsCon
     feature.check(holder, constParameter, "const generics")
 }
 
+private fun checkConstGenericsDefaults(holder: RsAnnotationHolder, default: RsExpr?) {
+    if (default == null) return
+    CONST_GENERICS_DEFAULTS.check(holder, default, "const generics defaults")
+}
+
+private fun checkConstArguments(holder: RsAnnotationHolder, args: List<RsExpr>) {
+    if (args.none { it is RsBinaryExpr }) return
+    val availability = CONST_GENERICS_DEFAULTS.availability(args.first())
```

Comment:
Why do we check this feature here at all? It looks suspicious because:
- we call this function not only for default values but also for const arguments (as says function name)
- compiler emits `expressions must be enclosed in braces to be used as const generic arguments` regardless of `const_generics_defaults` feature availability

Maybe just drop the corresponding check?